test/sql/cloud/hierarchical_namespace.test

```diff
@@ -0,0 +1,56 @@
+# name: test/sql/hierarchical_namespace.test
+# description: test azure extension with ADLS GEN2 storage
+# group: [azure]
+
+# Require statement will ensure this test is run with this extension loaded
+require azure
+
+require-env AZURE_STORAGE_DFS_STORAGE_NAME
+
+require-env AZURE_STORAGE_DFS_CONNECTION_STRING
+
+statement ok
+CREATE SECRET s1 (
+    TYPE AZURE,
+    CONNECTION_STRING '${AZURE_STORAGE_DFS_CONNECTION_STRING}'
+);
+
+query I
+SELECT count(*) FROM 'abfss://hn1/key1=m*/*.csv';
```

Comment:
This one fails for me with:

`Invalid Error: Fail to get a new connection for: ://. URL using bad/illegal format or missing URL`

I suspect i'm doing something wrong with the connection strings, same as for the  `scripts/prepare_hierarchical_namespace_test.sh` script